proto/ibc/applications/fee/v1/ack.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+
+package ibc.applications.fee.v1;
+import "gogoproto/gogo.proto";
+option go_package = "github.com/cosmos/ibc-go/modules/apps/29-fee/types";
+
+// IncentivizedAcknowledgement is the acknowledgement format to be used by applications wrapped in the fee middleware
+// It contains the raw acknowledgement bytes, as well as the forward relayer address
+message IncentivizedAcknowledgement {
+  bytes  result                  = 1;
```

Comment:
Is this the underlying apps acknowledgment bytes? If so wondering if this is the right naming. 

This will need to be wrapped again in the generic acknowledgment type, right? Maybe I'm not understanding it properly.

Reply:
just spoken to Aditya: confirmed that this does not need to be wrapped bc the version negotiation will be able differentiate which ack to use/return.